src/widget_type.h

```diff
@@ -460,6 +480,10 @@ class NWidgetPIPContainer : public NWidgetContainer {
 	uint8 pip_pre;            ///< Amount of space before first widget.
 	uint8 pip_inter;          ///< Amount of space between widgets.
 	uint8 pip_post;           ///< Amount of space after last widget.
+
+	uint8 uz_pip_pre;            ///< Amount of space before first widget.
```

Comment:
For NWidgetResizeBase you amended the uz comments with `Unscaled`, for this you did not. When you're changing these lines, maybe align the comments with the ones above it as well?

Reply:
Also added comments for the other variables.